<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ELoad_SettingToolWid</class>
 <widget class="QWidget" name="ELoad_SettingToolWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="serialBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>86</y>
     <width>93</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>串口设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>450</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>时间间隔:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="timeBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>106</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>500ms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000ms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1500ms</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cmdBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>单命令</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双命令</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="devNumBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>221</y>
     <width>93</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="serialLab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>52</y>
     <width>133</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>COM0  未打开</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="timeBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>492</y>
     <width>93</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>357</y>
     <width>93</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>180</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>级联数量:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>310</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>命令模式:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>180</y>
     <width>52</width>
     <height>33</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>33</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
